src/main/resources/view/LightTheme.css

```
@@ -222,7 +218,7 @@
     -fx-border-width: 2;
     -fx-background-radius: 0;
     -fx-background-color: #F7ECDE;
-    -fx-font-family: "Segoe UI", Helvetica, Arial, sans-serif;
+    -fx-font-family: "Courier New", Helvetica, Arial, sans-serif;
```

this should me `monospace` if you want a fallback
```suggestion
    -fx-font-family: "Courier New", monospace;
```
probably the same for the other font things